paths/job_template_locales/show.yaml

```
@@ -0,0 +1,54 @@
+---
+summary: Get a single job template locale
+description: Get a single job template locale for a given job template.
```

As I can see it's not possible to get/create/delete/update without project_id in `Job Template Locales` section without `project_id`.

Would we like to add this in description e.g.
```suggestion
description: Get a single job template locale for a given project and job template.
```

what do you think?